projects/plugins/jetpack/modules/subscriptions/subscribe-modal/subscribe-modal.js

```diff
@@ -0,0 +1,27 @@
+window.onload = function () {
```

Comment:
There's a GB package for this: https://github.com/WordPress/gutenberg/tree/trunk/packages/dom-ready#readme

It has a bit more complete version of onload:

https://github.com/WordPress/gutenberg/blob/trunk/packages/dom-ready/src/index.js#L31-L45

Reply:
Added this.